src/models/api/README.md

```
@@ -1,5 +1,7 @@
 # List of source files stored in `src/models/api` directory
 
 ## [__init__.py](__init__.py)
-Typed HTTP API models (OpenAPI-oriented) for FastAPI routes.
+Typed HTTP API models (OpenAPI-oriented) for FastAPI routes. Exposes the [`responses`](responses/README.md) subpackage.
 
+## [responses/](responses/README.md)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)